### PR TITLE
Feature achievements - screen added, no achievements yet.

### PR DESCRIPTION
**Pull request by jessearens**
This code adds the achievement screen to the game. Later, achievements will be shown here.
I still have to figure out how to fix the GUI test for the Achievement screen. Somehow it does not allow me to extend the GuiTest class into the TestAchievementScreen class.

### CODE REVIEW

File: src/test/java/com/github/fishio/gui/TestAchievementScreen.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.github.fishio.gui;
```
Comment (jessearens) [inline src/test/java/com/github/fishio/gui/TestAchievementScreen.java:1]: If somebody has any ideas to why this class does not recognize GuiTest when extending, be my guest and explain it to me. >bug

Comment (Taeir) [inline src/test/java/com/github/fishio/gui/TestAchievementScreen.java:1]: There is a mistake in switchToScreen, the screen is called "achievementScreen", not "AchievementScene".
Also, pull request #64 makes some GuiTest changes that makes GuiTests more stable, which might prevent other problems.

**Comment by Diocruel** [comment]
As there are problems we should close the branch for now.

**Comment by jessearens** [comment]
Improved the code by merging the pull request #64 and making some more changes.
Collateral damage is that the TestAchievementScreen also works now.
<testing,feature,bug

**Comment by Taeir** [comment]
Looks good!

<testing,feature,bug

**Comment by Diocruel** [comment]
Agreed, I will merge this into my branch as well and add a logging in the correct places.

Merged.

<testing,feature,bug

**Pull request merged**